plugins/module_utils/kibana.py

```diff
@@ -182,12 +182,12 @@ def get_pkg_policy(self,integration_name, agent_policy_id):
         pkg_policy_object = pkgPolicy
     return pkg_policy_object
   
-  def create_pkg_policy(self,pkg_policy_name, pkg_policy_desc, agent_policy_id, integration_object):
+  def create_pkg_policy(self,pkg_policy_name, pkg_policy_desc, agent_policy_id, integration_object, namespace="default"):
```

Comment:
Since namespace is set to default to "default" in both modules, I think it's better to drop the default argument here.

Reply:
That was simply so you didn't have to pass it.  I didn't want to require it for future use.